### DISCUSSION TITLE
Add theme policy to docs

### DISCUSSION THREAD

**Discussion by diktomat** [helix-editor/helix#10141] [General]
Currently, the docs on creating a theme only point a link to user-submitted themes in `runtime/themes`. This suggests one can submit ones own theme there also. Is there a policy on what themes get shipped first party with Helix? Could get out of hand easily when everybody submits their own theme to the point there are hundreds of them in the repo. That's why I haven't submitted [my port](https://github.com/d12bb/dotfiles/blob/main/helix/themes/gruvbox_baby.toml) of [gruvbox-baby](https://github.com/luisiacc/gruvbox-baby) yet, we already have gruvbox and gruvbox_light in there which are kinda similar.

1. If there is such policy, I think it should be added to the docs or as pull request template here.
2. If there isn't, I think one should be created and added (1.) after.

**Comment by the-mikedavis** [comment]
Currently there isn't a rigorous policy - we tend to accept any theme contributions without considering if they're 'too close' to an existing theme (for example, tokyo_night and tokyo_night_storm).

I'm hesitant to come up with a rigorous policy and put it into the documentation because the current strategy of centralizing community themes in `helix-editor/helix` probably shouldn't continue in the long run - the maintenance burden is small but non-trivial (see the new/updated themes sections in the changelog).

We could split out the themes into a new `helix-editor/themes` repo to keep them centralized but in the long-run I'd like to see a community-run theme repo like https://github.com/CptPotato/helix-themes take off. We could probably do more in this repo to advertise those. I wouldn't complain if in the end `helix-editor/helix` only contained the base16 themes and the default (`theme.toml` - the purple one), plus we could go fetch the separate theme repo when building releases so that releases still have many built-in themes.
